README.md

```diff
@@ -12,21 +12,21 @@ The short version is to run these commands:
 ```sh
 tsc -p path/to/tsconfig.json --generateTrace traceDir
 npm install --no-save @typescript/analyze-trace
-npx analyze-trace traceDir
+npx @typescript/analyze-trace traceDir
```

Comment:
This change is technically not correct; the previous command installs `@typescript/analyze-trace` into `node_modules`, placing `analyze-trace`, `print-trace-types`, and `simplify-trace-types` into `node_modules/.bin`. That means that the old npx command is correct.

That being said, I am not against this change so long as we also remove instructions on how to install the package and just use `npx` everywhere, but this is problematic because the package contains three commands (and it's not clear to me which one of the three is going to be chosen as the "main" binary when run with npx?)

Reply:
Makes sense. In that case we can keep the old command only. Thanks for the clarification. I'll close the PR.